public/app/ui/pages/detail/DetailPage.js

```diff
@@ -28,9 +28,10 @@
                     return !!this.data();
                 }.bind(this));
 
-                this.typeHeader = ko.pureComputed(function () {
-                    return settings.collectionName + ' Record';
-                });
+                this.collectionName = ko.pureComputed(function () {
+                    var data = this.data();
+                    return data ? data.type : undefined;
```

Comment:
Use the type name from the data; this is consistent with what the search displays.